### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by Vitineth** [Vitineth/st-andrews-covid#16]
**Describe the solution you'd like**
A set of features should be implemented to improve general site accessibility:
* [x] Increase the size of links (avoid one word links like 'here')
* [x] Include email in text, not just as a mailto link
* [x] Include either semantic headers or aria labels to the important info and updates sections
* [x] Include aria labels on all relevant components
* [x] Link the tabs to the content in a meaningful way
* [x] Make tabs act semantically like radio buttons

**Comment by Vitineth** [comment]
Should be resolved by #17
